vault/logical_system.go

```diff
@@ -2234,10 +2266,24 @@ func (b *SystemBackend) handleConfigUIHeadersDelete(ctx context.Context, req *lo
 	return nil, nil
 }
 
+type (
+	RawBackend struct {
```

Comment:
Can we make a new file for the raw backend?

Reply:
Sure

Reply:
Done.